pkg/action/programkind.go

```diff
@@ -16,31 +16,40 @@ import (
 	"github.com/liamg/magic"
 )
 
+var archiveMap = map[string]bool{
+	".apk":    true,
+	".jar":    true,
+	".tar":    true,
+	".tar.gz": true,
+	".tar.xz": true,
```

Comment:
wonder if should also add .tgz here as well?

Reply:
Added in [`424a907` (#144)](https://github.com/chainguard-dev/bincapz/pull/144/commits/424a907f4bbc7201f6f4bc242c2e2b0ac06ec853).